.php_cs.php

```diff
@@ -3,7 +3,7 @@
 use PhpCsFixer\Config;
 
 $header = <<<'EOT'
-This file was generated by the dh-core/api-client-generator.
+This file was generated by DoclerLabs/api-client-generator.
```

Comment:
Composer package name is docler-labs/api-client-generator, lets put it here like that.

Reply:
can we rename the github repo as well? It is a bit confusing anyway now.

Reply:
Do you mean to change the organization name or composer package name?

Reply:
Let's rename composer package to `docler/api-client-generator`? Postman follows this convention. Github organization is PostmanLabs, but composer package is postman/something.

Reply:
I'm ok with docler-labs to be honest. docler/api-client-generator is also fine. If you think that way it would be much less confusing, could you please change it in this repo and api-calient-base?

Reply:
PR updated with `docler-labs/api-client-generator` - current package name - the easiest way to go.